### PR TITLE
add as doc

### PR DESCRIPTION
**Pull request by takayuki5168**
## Description

add README of as (only pacmod_interface)

## Pre-Review Checklist for the PR Author

**PR Author should check the checkboxes below when creating the PR.**

- [ ] Read [pull request guidelines][pull-request-guidelines]
- [ ] Code follows [coding guidelines][coding-guidelines]
- [ ] Assign PR to reviewer

## Checklist for the PR Reviewer

**Reviewers should check the checkboxes below before approval.**

- [ ] Commits are properly organized and messages are according to the guideline
- [ ] Code follows [coding guidelines][coding-guidelines]
- [ ] (Optional) Unit tests have been written for new behavior
- [ ] PR title describes the changes

## Post-Review Checklist for the PR Author

**PR Author should check the checkboxes below before merging.**

- [ ] All open points are addressed and tracked via issues or tickets
- [ ] Write [release notes][release-notes]

## CI Checks

- **Build and test for PR / build-and-test-pr**: Required to pass before the merge.
- **Build and test for PR / clang-tidy-pr**: NOT required to pass before the merge. It is up to the reviewer(s). Found false positives? See the [guidelines][clang-tidy-guidelines].
- **Check spelling**: NOT required to pass before the merge. It is up to the reviewer(s). See [here][spell-check-dict] if you want to add some words to the spell check dictionary.

[clang-tidy-guidelines]: https://tier4.github.io/autoware.proj/tree/main/developer_guide/ClangTidyGuideline/
[coding-guidelines]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/1194394777/T4
[pull-request-guidelines]: https://tier4.github.io/autoware.proj/tree/main/developer_guide/PullRequestGuideline/
[release-notes]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/563774416
[spell-check-dict]: https://github.com/tier4/autoware-spell-check-dict#how-to-contribute

### CODE REVIEW

**Comment by tkimura4** [comment]
nice!!!!!!

File: vehicle/as/README.md [hunk 1]
```diff
@@ -1,15 +1,82 @@
...
+  | `/pacmod/parsed_tx/steer_rpt`       | pacmod_msgs::msg::SystemRptFloat | current steering wheel angle                                            |
+  | `/pacmod/parsed_tx/wheel_speed_rpt` | pacmod_msgs::msg::WheelSpeedRpt  | current wheel speed                                                     |
+  | `/pacmod/parsed_tx/accel_rpt`       | pacmod_msgs::msg::SystemRptFloat | current accel pedal                                                     |
+  | `/pacmod/parsed_tx/brake_rpt`       | pacmod_msgs::msg::SystemRptFloat | current brake pedal                                                     |
+  | `/pacmod/parsed_tx/shift_rpt`       | pacmod_msgs::msg::SystemRptInt   | current gear status                                                     |
+  | `/pacmod/parsed_tx/turn_rpt`        | pacmod_msgs::msg::SystemRptInt   | current turn indicators status                                          |
+  | `/pacmod/parsed_tx/global_rpt`      | pacmod_msgs::msg::GlobalRpt      | current status of other parameters (e.g. override_active, can_time_out) |
```
Comment (tkimura4) [inline vehicle/as/README.md:33]: pacmod_msgs -> pacmod3_msgs

https://github.com/tier4/AutowareArchitectureProposal.iv/pull/610

Comment (takayuki5168) [reply vehicle/as/README.md]: https://github.com/tier4/AutowareArchitectureProposal.iv/pull/616/commits/3ab02480901b4554d773a6039188106abb94f744
Done

File: vehicle/as/README.md [hunk 2]
```diff
@@ -1,15 +1,82 @@
...
+
+### ROS Parameters
+
+| Name                              | Type   | Description                                                                               |
+| --------------------------------- | ------ | ----------------------------------------------------------------------------------------- |
+| `base_frame_id`                   | string | frame id (assigned in pacmod command, but it does not make sense)                         |
+| `command_timeout_ms`              | double | timeout [ms]                                                                              |
+| `loop_rate`                       | double | loop rate to publish commands                                                             |
+| `steering_offset`                 | double | steering wheel angle offset                                                               |
+| `enable_steering_rate_control`    | bool   | when enabled, max steering wheel rate is used for steering wheel angluar velocity command |
```
Comment (tkimura4) [inline vehicle/as/README.md]:
```suggestion
| `enable_steering_rate_control`    | bool   | when enabled, max steering wheel rate is used for steering wheel angular velocity command |
```

File: vehicle/as/README.md [hunk 3]
```diff
@@ -1,15 +1,82 @@
...
+  | `pacmod/as_rx/raw_steer_cmd` | pacmod3_msgs::msg::SteerSystemCmd | raw steering wheel angle and angular velocity command |
+
+- To Autoware
+
+  | Name                               | Type                                                   | Description                                          |
+  | ---------------------------------- | ------------------------------------------------------ | ---------------------------------------------------- |
+  | `/vehicle/status/control_mode`     | autoware_auto_vehicle_mgs::msg::ControlModeReport      | control mode                                         |
+  | `/vehicle/status/twist`            | autoware_auto_vehicle_mgs::msg::VelocityReport         | velocity                                             |
+  | `/vehicle/status/steering`         | autoware_auto_vehicle_mgs::msg::SteeringReport         | steering wheel angle                                 |
+  | `/vehicle/status/gear_cmd`         | autoware_auto_vehicle_mgs::msg::GearReport             | gear status                                          |
```
Comment (tkimura4) [inline vehicle/as/README.md:54]: Could you change the topic name to "/vehicle/status/shift" ?
(Change the code as well)

Comment (tkimura4) [reply vehicle/as/README.md:54]: Also, control/gear_cmd -> control/shift_cmd

Comment (takayuki5168) [reply vehicle/as/README.md:54]: In the P/C sync mtg (currently conducted), it was decided that we use not shift but gear for topic names. Is it ok?

Comment (tkimura4) [reply vehicle/as/README.md:54]: OK!!
 (/vehicle/status/gear_cmd  ? /vehicle/status/gear? )

Comment (takayuki5168) [reply vehicle/as/README.md:54]: Thanks, this one. `/vehicle/status/gear_cmd`

Comment (tkimura4) [reply vehicle/as/README.md:54]: Thn, could you create the PR to change shift -> gear of other packages?

Comment (takayuki5168) [reply vehicle/as/README.md:54]: Alright. Sugahara-san is working on some packages and will replace those message name, and I will work on other packages.

Comment (takayuki5168) [reply vehicle/as/README.md:54]: Ah but now I found that gear_cmd for GearReport is a bit weird...

Comment (TakaHoribe) [reply vehicle/as/README.md:54]: Please going with `/vehicle/status/gear`!

**Comment by takayuki5168** [comment]
After https://github.com/tier4/AutowareArchitectureProposal.iv/pull/640, this PR will be merged. There is still something to be fixed in this PR though.

File: vehicle/as/README.md (hunk 3 above)
Comment (takayuki5168) [reply vehicle/as/README.md:54]: We decided to use formats like `/vehicle/status/gear_status`

**Comment by tkimura4** [comment]
こちら、トピック名
https://github.com/tier4/AutowareArchitectureProposal.iv/pull/646
にあわせて変更したらマージしましょうか

**Comment by takayuki5168** [comment]
@tkimura4 @taikitanaka3 Could you

File: vehicle/pacmod_interface/README.md [hunk 4]
```diff
@@ -1,15 +1,80 @@
...
+  | `/control/command/hazard_lights_cmd`   | autoware_auto_vehicle_msgs::msg::HazardLightsCommand     | hazard lights command                                 |
+  | `/vehicle/engage`                      | autoware_auto_vehicle_msgs::msg::Engage                  | engage command                                        |
+  | `/vehicle/command/actuation_cmd`       | autoware_vehicle_msgs::msg::ActuationCommandStamped      | actuation (accel/brake pedal, steering wheel) command |
+  | `/control/command/emergency_cmd`       | autoware_vehicle_msgs::msg::VehicleEmergencyStamped      | emergency command                                     |
+
+- From Pacmod
+
+  | Name                                | Type                              | Description                                                             |
+  | ----------------------------------- | --------------------------------- | ----------------------------------------------------------------------- |
+  | `/pacmod/parsed_tx/steer_rpt`       | pacmod3_msgs::msg::SystemRptFloat | current steering wheel angle                                            |
```
Comment (taikitanaka3) [inline vehicle/pacmod_interface/README.md:25]: due to current change pacmod3 parse tx and parse rx is removed and it is devided by hoge_rpt and hoge_cmd

Comment (takayuki5168) [reply vehicle/pacmod_interface/README.md:25]: @taikitanaka3 could you tell me which PR the change is included?

Comment (takayuki5168) [reply vehicle/pacmod_interface/README.md]: @taikitanaka3 how about this one https://github.com/tier4/AutowareArchitectureProposal.iv/pull/616/commits/d96d76dd1595615be4c5307c947f247e50720255

**Review by taikitanaka3 — APPROVED** [review]
LGTM

**Pull request merged**